src/data/symbol/symbol.tsv

```diff
@@ -976,6 +1074,7 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	㍕	れむ たんい たんい	単位		OTHER		U+3355
 記号	㍖	れんとげん たんい	単位		OTHER		U+3356
 記号	㍗	わっと たんい	単位		OTHER		U+3357
+記号	🈀	ほか			OTHER		U+1F200
```

Comment:
It seems some platform (i.e. macOS) have not supported a font for this character yet.
Since we need an extra treatment for those characters, would you remove this line from this PR?

Reply:
I'll revert its addition.